pandas/tests/reshape/test_melt.py

```diff
@@ -289,34 +289,53 @@ def test_melt_missing_columns_raises(self):
         df = pd.DataFrame(np.random.randn(5, 4), columns=list("abcd"))
 
         # Try to melt with missing `value_vars` column name
-        msg = "The following '{Var}' are not present in the DataFrame: {Col}"
-        with pytest.raises(
-            KeyError, match=msg.format(Var="value_vars", Col="\\['C'\\]")
-        ):
+        msg = r"The following '{var}' are not present in the DataFrame: \[{col}\]"
```

Comment:
Made a couple small cleanups to the error message not directly related to the bug in question:
- Added brackets around `{col}` so they didn't need to be manually added to each message, reducing a few multi-line statements to a single line
- Made the variables lowercase to be more stylistically consistent with other usages of `format`